pkg/csi/service/common/util.go

```diff
@@ -354,6 +355,7 @@ func ConvertVolumeHealthStatus(volHealthStatus string) (string, error) {
 		// This implies the volume does not exist any more.
 		// Set health annotation to "Inaccessible" so that
 		// the caller can make appropriate reactions based on this status
+		log.Infof("SPBM returns undefined health status %q", volHealthStatus)
```

Comment:
The log message contradicts with the comment above.

Reply:
Yes. It will keeps logging if SPBM returns undefined health status, which is empty string. This will happen if the volume has been removed from FCD inventory, in that case, SPBM cannot populate volume health status for that volume, and thus we got empty string. See this https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/852/files.
Move this log to DEBUG level.